cli/graph.rs

```diff
@@ -1030,12 +1030,13 @@ mod tests {
     );
     let h = handler.borrow();
     assert_eq!(h.cache_calls.len(), 1, "only one file should be emitted");
+    // FIXME(bartlomieju): had to add space in `<div>`, probably a quirk in swc_ecma_codegen
     assert!(
       h.cache_calls[0]
         .2
         .to_string()
         .unwrap()
-        .contains("<div>Hello world!</div>"),
+        .contains("<div >Hello world!</div>"),
```

Comment:
Little quirk I encountered; but it should still be ok to land.

Full output after transform:
```
export default class A {
    render() {
        return <div >Hello world!</div>;
    }
};
//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIjxodHRwczovL2Rlbm8ubGFuZC94L3RyYW5zcGlsZS50c3g+Il0sInNvdXJjZXNDb250ZW50IjpbImV4cG9ydCBkZWZhdWx0IGNsYXNzIEEge1xuICByZW5kZXIoKSB7XG4gICAgcmV0dXJuICg8ZGl2PkhlbGxvIHdvcmxkITwvZGl2Pik7XG4gIH1cbn1cbiJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoicUJBQUEsQ0FBQTtBQUNBLFVBQUE7Z0JBQ0EsR0FBQSxFQUFBLFlBQUEsRUFBQSxHQUFBIn0=
```

CC @kdy1 @kitsonk

Reply:
https://github.com/swc-project/swc/blob/ea091337d80760df6e09bce90ed3906ecb3e7626/ecmascript/codegen/src/jsx.rs#L26 

This line is problematic.